### PR TITLE
Fix vim mode error

### PR DESCRIPTION
**Pull request by jeremyrsmith**

### CODE REVIEW

File: polynote-frontend/polynote/cell.js [hunk 1]
```diff
@@ -985,16 +984,13 @@ export class CodeCell extends Cell {
...
             delete this.vim;
         }
         this.hideVim();
     }
 
     hideVim() {
-        if (this.statusLine && !this.statusLine.contains(document.activeElement)) {
-            this.statusLine.classList.toggle('hide', true);
-        }
+        this.statusLine.innerHTML = '';
```
Comment (indig) [inline polynote-frontend/polynote/cell.js]: We no longer need the `if (this.statusLine)` guard?

File: polynote-frontend/polynote/cell.js [hunk 2]
```diff
@@ -969,12 +969,11 @@ export class CodeCell extends Cell {
...
             if (!this.vim) {
-                console.log("init vim for cell", this.id)
                 if (!this.statusLine) {
                     this.statusLine = div(["vim-status"], []);
                 }
                 this.vim = createVim(this.editor, this.statusLine);
-                this.cellInput.querySelector(".cell-footer").insertBefore(this.statusLine, this.execInfoEl);
+                this.cellInput.querySelector(".cell-footer").appendChild(this.statusLine);
             }
             this.statusLine.classList.toggle('hide', false);
```
Comment (indig) [inline polynote-frontend/polynote/cell.js]: Did you want to change this to `.remove` as we discussed?

File: polynote-frontend/polynote/cell.js (hunk 1 above)
Comment (jeremyrsmith) [reply polynote-frontend/polynote/cell.js]: I'm not convinced it was ever needed, because `this.statusLine` is always created in the `Cell` constructor.

Comment (indig) [reply polynote-frontend/polynote/cell.js]: Oh I see, looks like the guard was left over from the initial implementation which didn't create it in the constructor. Nice! :)

**Pull request merged**